fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/facades/SimulationFacade.java

```diff
@@ -323,19 +346,16 @@ public final double getLengthOfLane(String edgeId, int laneIndex) throws Interna
      * Enables the calculation of distance sensor values for the given vehicle.
      */
     public void configureDistanceSensors(String vehicleId, double maximumLookahead, boolean front, boolean rear) {
+        SumoVehicleState vehicleState = getVehicleState(vehicleId);
         if (front) {
-            if (maximumLookahead > 0) {
-                vehiclesWithFrontSensor.put(vehicleId, maximumLookahead);
-            } else {
-                vehiclesWithFrontSensor.remove(vehicleId);
-            }
+            vehicleState.frontSensorDistance = maximumLookahead > 0 ? maximumLookahead : null;
         }
         if (rear) {
-            if (maximumLookahead > 0) {
-                vehiclesWithBackSensor.put(vehicleId, maximumLookahead);
-            } else {
-                vehiclesWithBackSensor.remove(vehicleId);
-            }
+            vehicleState.rearSensorDistance = maximumLookahead > 0 ? maximumLookahead : null;
+        }
+        this.noRearSensorConfigured = true;
+        for (SumoVehicleState aState : sumoVehicles.values()) {
```

Comment:
This seems a little bit expensive. But I suppose it's fine, as there usually aren't many Sensor activations

Reply:
I could actually get rid of the whole calculation of follower distance, as SUMO already allows to retrieve the follower via TraCI and libsumo. :) I changed this on the other branch however